src/Map/TypedMapInterface.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types=1);
```

Comment:
Should `getKeyType()` be hinted to return a `string`?

Reply:
I think it could be on getKeyType as well as getValueType and string can be on the constructor of TypedMap. It's doc-blocked as a string in there and on the properties, but it was mixed return for the getValueType. Pushing up this change (with rebase) shortly.